From e4792b00c88103bba9239bb5cabb10b95c4ae531 Mon Sep 17 00:00:00 2001
From: Nicholas Corgan <nick.corgan@ettus.com>
Date: Fri, 4 Mar 2016 10:32:32 -0800
Subject: [PATCH 26/27] Fixed minor warnings:

* nirio_driver_iface_win: labeled unused variable for MinGW builds
* b200_impl: fixed unreferenced variable warning
* n200_image_loader: fixed signed vs. unsigned comparison
---
 host/lib/transport/nirio/nirio_driver_iface_win.cpp | 4 ++--
 host/lib/usrp/b200/b200_impl.cpp                    | 2 +-
 host/lib/usrp/usrp2/n200_image_loader.cpp           | 4 ++--
 3 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/host/lib/transport/nirio/nirio_driver_iface_win.cpp b/host/lib/transport/nirio/nirio_driver_iface_win.cpp
index 7179231..c8484e9 100644
--- a/host/lib/transport/nirio/nirio_driver_iface_win.cpp
+++ b/host/lib/transport/nirio/nirio_driver_iface_win.cpp
@@ -1,5 +1,5 @@
 //
-// Copyright 2013,2015 Ettus Research LLC
+// Copyright 2013,2015-2016 Ettus Research LLC
 //
 // This program is free software: you can redistribute it and/or modify
 // it under the terms of the GNU General Public License as published by
@@ -66,7 +66,7 @@ nirio_status rio_ioctl(
             read_buf, static_cast<DWORD>(read_buf_len),
             &outLen, &zeroedOverlapped )))
     {
-        int_fast32_t lastError = GetLastError();
+        UHD_UNUSED(int_fast32_t lastError) = GetLastError();
         return NiRio_Status_SoftwareFault;
     }
 
diff --git a/host/lib/usrp/b200/b200_impl.cpp b/host/lib/usrp/b200/b200_impl.cpp
index 200d99a..18dcdad 100644
--- a/host/lib/usrp/b200/b200_impl.cpp
+++ b/host/lib/usrp/b200/b200_impl.cpp
@@ -348,7 +348,7 @@ b200_impl::b200_impl(const uhd::device_addr_t& device_addr, usb_device_handle::s
                 handle = dev_handle;
                 break;
             }
-        } catch (const uhd::exception &e) { continue; }
+        } catch (const uhd::exception&) { continue; }
     }
     UHD_ASSERT_THROW(handle.get() != NULL); //better be found
 
diff --git a/host/lib/usrp/usrp2/n200_image_loader.cpp b/host/lib/usrp/usrp2/n200_image_loader.cpp
index 29bec8b..d6acfe3 100644
--- a/host/lib/usrp/usrp2/n200_image_loader.cpp
+++ b/host/lib/usrp/usrp2/n200_image_loader.cpp
@@ -1,5 +1,5 @@
 //
-// Copyright 2015 Ettus Research LLC
+// Copyright 2015-2016 Ettus Research LLC
 //
 // This program is free software: you can redistribute it and/or modify
 // it under the terms of the GNU General Public License as published by
@@ -201,7 +201,7 @@ static UHD_INLINE bool n200_response_matches(const n200_fw_update_data_t *pkt_in
                                              n200_fw_update_id_t pkt_code,
                                              size_t len){
     return (len > offsetof(n200_fw_update_data_t, data) and
-            ntohl(pkt_in->id) == pkt_code);
+            ntohl(pkt_in->id) == (unsigned)pkt_code);
 }
 
 static uhd::device_addr_t n200_find(const image_loader::image_loader_args_t &image_loader_args){
-- 
2.1.4

